id/about/index.md

```diff
@@ -169,27 +167,27 @@ digunakan pada setiap member instan `self.`
 
 Ruby kaya fitur, antara lain sebagai berikut:
 
-* Ruby memiliki fitur-fitur yang menangani exception, seperti Java atau
-  Python, untuk mempermudah menangani error.
+* Ruby memiliki fitur-fitur yang menangani *exception*, seperti Java atau
+  Python, untuk mempermudah menangani *error*.
 
-* Ruby menyediakan mark-and-sweep garbage collector untuk semua obyek
-  Ruby. Tidak perlu me-maintain reference count pada library extension.
+* Ruby menyediakan *mark-and-sweep garbage collector* untuk semua objek
+  Ruby. Tidak perlu me-*maintain reference count* pada *extension*.
```

Comment:
should it be menjaga or mempertahankan instead of me-maintain?

Reply:
I prefer "menjaga". What do you think?

Reply:
Sounds good 👍

Reply:
I just updated it to "menjaga".